backend/inspirehep/mailing/api/jobs.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from elasticsearch_dsl.query import Q
+from flask import current_app, render_template
+
+from inspirehep.search.api import JobsSearch
+
+from ..providers.mailchimp import create_mailchimp_campaign, send_mailchimp_campaign
+
+
+def get_jobs_from_last_week():
+    """Jobs created the last 7 days."""
+    query = Q("range", **{"_created": {"gte": "now-7d/d", "lt": "now/d"}})
+    search = JobsSearch().query(query).sort("_created")
```

Comment:
Just to make sure, is this the order we want the jobs to be sorted by? I know the task mentions ordered by date posted (and doesn't say ascending or descending).But in the example they seem to be descending.(so - _created)